### PR TITLE
Add autonomous car dev assignment

### PR DESCRIPTION
**Pull request by JakeDeFranco**
Add the Autonomous Car Dev Assignment to the Server

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,22 @@
 # Interview Test Server
 
-For the Points developer take-home assignment, the following task is required: you are to build a tax calculator against this API. The purpose is to develop an income tax calculator to help illustrate how marginal taxes work. This server is pretty simple, only returning the tax rates for a given year. You are to calculate the amount of taxes owed by inputting a yearly salary, then displaying the different rates, the total amount owed and the effective tax rates.
+For the Points developer take-home assignment, the following task is required: you are to build a tax calculator against
+ this API. The purpose is to develop an income tax calculator to help illustrate how marginal taxes work. This server is
+ pretty simple, only returning the tax rates for a given year. You are to calculate the amount of taxes owed by 
+ inputting a yearly salary, then displaying the different rates, the total amount owed and the effective tax rates.
```
Comment (JakeDeFranco) [inline README.md:6]: Nothing changed here, except now each line is <= 120 chars

File: README.md [hunk 2]
```diff
@@ -1,8 +1,22 @@
...
 
 ## Instructions
+For all of the below assignments there are different levels that you can work on:
+ * **Level 1:** The easier level, the API always behaves and the response never changes.
+ * **Level 2:** More difficult, the API doesn't always behave and includes gotchas.
+ 
+If you complete the first level with ease, feel free to jump to the next one. 
+
+**Please timebox your solution to one day total.**
```
Comment (JakeDeFranco) [inline README.md:15]: Moved this section up here since now there are two assignments.

File: templates/base.html [hunk 3]
```diff
@@ -7,8 +7,7 @@
             font-family: Arial, Helvetica, sans-serif;
             background-color: #efefef;
             width: 100%;
-            height:100%;
-            overflow: hidden;
+            height: 100%;
```
Comment (JakeDeFranco) [inline templates/base.html:10]: The autonomous car instructions page is a bit larger due to new description and instructions sections as well as now offering multiple endpoints for level 1. 

`overflow: hidden;` would make scrolling down impossible.

File: README.md [hunk 4]
```diff
@@ -12,18 +26,72 @@ For the Points developer take-home assignment, the following task is required: y
     * Displays the amount of taxes owed per band
     * Displays the effective rate
 
-There are different levels to this assignment, level 1 being the easiest, level 2 having some gotchas, etc. If you complete a level with ease, feel free to jump to the next one. Please timebox your solution to one day total.
+### Level 1
+
+* Fetch the tax rates from this server at: [localhost:5000/tax-calculator/brackets](http://localhost:5000/tax-calculator/brackets)
+
+### Level 2
```
Comment (pts-jweatherby) [inline README.md]: This was updated to look like: 
**Level 2** _(sometimes returns bad responses, handle accordingly)_

File: README.md [hunk 5]
```diff
@@ -12,18 +26,72 @@ For the Points developer take-home assignment, the following task is required: y
...
+
+* Fetch the tax rates by year: [localhost:5000/tax-calculator/brackets/<2019,2020...>](http://localhost:5000/tax-calculator/brackets/2020)
+
+_Warning: this endpoint has gotchas_
```
Comment (pts-jweatherby) [inline README.md]: this was removed in the latest changes

File: README.md [hunk 6]
```diff
@@ -12,18 +26,72 @@ For the Points developer take-home assignment, the following task is required: y
...
+   will return a route object that should result in a successful simulation, **with zero obstacles on the track**.
+ * [/autonomous-car/routes/success-with-obstacles](http://localhost:5000/autonomous-car/routes/success-with-obstacles) 
+   will return a route object that should result in a successful simulation, **with obstacles on the track**.
+ * [/autonomous-car/routes/failure-out-of-bounds](http://localhost:5000/autonomous-car/routes/failure-out-of-bounds) 
+   will return a route object that should result in failure due to running out of bounds.
+ * [/autonomous-car/routes/failure-hits-obstacle](http://localhost:5000/autonomous-car/routes/failure-hits-obstacle) 
+   will return a route object that should result in failure due to running into an obstacle.
+
 
 ### Level 2
```
Comment (pts-jweatherby) [inline README.md]: Same here
**Level 2** _(sometimes returns bad responses, handle accordingly)_

File: README.md [hunk 7]
```diff
@@ -12,18 +26,72 @@ For the Points developer take-home assignment, the following task is required: y
...
 
-* Fetch the tax rates by year: http://localhost:5000/tax-calculator/brackets/<2019|2020|...>
+For this level, you can fetch a random route (including the empty-route) using the endpoint:
+* [/autonomous-car/routes/random](http://localhost:5000/autonomous-car/routes/random)
 
 _Warning: this endpoint has gotchas_
```
Comment (pts-jweatherby) [inline README.md]: and to remove as well

File: README.md (hunk 4 above)
Comment (JakeDeFranco) [reply README.md]: Hmm doesn't look like the master branch has this change? I'll make these changes though.

File: README.md (hunk 7 above)
Comment (JakeDeFranco) [reply README.md]: Updated 👍

File: README.md (hunk 5 above)
Comment (JakeDeFranco) [reply README.md]: Updated 👍

File: README.md (hunk 4 above)
Comment (JakeDeFranco) [reply README.md]: Updated 👍

File: README.md (hunk 6 above)
Comment (JakeDeFranco) [reply README.md]: Updated 👍

File: README.md (hunk 7 above)
Comment (JakeDeFranco) [reply README.md]: Updated 👍

**Review by pts-jweatherby — APPROVED** [review]
Looks good, thanks for all the work! 🎉

**Pull request merged**